src/tools/e2e/generateTestData.ts

```diff
@@ -0,0 +1,29 @@
+import '../scriptInit'
+
+import * as path from 'path'
+
+import { MetadataController } from 'server/controller/metadata'
+import { DB } from 'server/db/db'
+import { Logger } from 'server/utils/logger'
+
+import { _writeTableToFile } from './_writeTableToFile'
+
+const OUTPUT_DIR = path.join(__dirname, 'fixtures')
+
+const exec = async (): Promise<void> => {
```

Comment:
The e2e tests , should only import , not export .
The export is a one time operation done by us when we need to update the data .

For now I would just create a tool (not e2e) that exports an assessment (now metadata , later also data)